tbx/project_styleguide/templates/patterns/pages/work/work_page_base.html

```diff
@@ -0,0 +1,108 @@
+            'author-title' : '{{ page.first_author.role }}',
+            'author-name' : '{{ page.first_author.name }}',
+            'published' : '{{ page.first_published_at|date:"Ymd" }}',
+            'tags' : [{% for tag in page.related_services.all %}'{{tag.name}}'{% if not forloop.last %}, {% endif %}{% endfor %}],
+            'read-time' : '{{ page.read_time }}',
+            'id': '{{ page.id }}',
+            'page-h1' : '{{ page.title }}',
+            'page-title' : '{{ page.title }} | Torchbox',
+            'page-path': '{{ page.slug }}',
+            'event': 'page-information'
+        });
+    </script>
+    <meta name="author" content="{{ page.first_author.name }}">
+    <meta property="og:article:published_time" content="{{ page.date|date:'Y-m-d' }}" />
+    {{ block.super }}
+
+{% endblock %}
+
+{% block content %}
+
+    <div class="work-hero grid grid--spacer-large">
+        <div class="work-hero__left">
+            <h1 class="heading heading--one-b mb-spacerMini md:mb-spacerSmall">{{ page.title }}</h1>
+
+            {% block intro %}
+                {# intro content goes here - different for work page and historical work page #}
+            {% endblock %}
+        </div>
+
+        <div class="work-hero__right">
+            {% if page.logo %}
+                {% image page.logo width-175 format-webp class="work-hero__logo" %}
+            {% endif %}
+
+            <p class="work-hero__client">
+                {{ page.client }}
+            </p>
+
+            <p class="work-hero__meta">
+                {% if page.date %}
+                    <time datetime="{{ page.date|date:'Y-m-d' }}">{{ page.date }}</time>
+                {% endif %}
+
+                {% if page.read_time and page.date %}
+                    <span aria-hidden="true" class="work-hero__divider">/</span>
+                {% endif %}
+
+                {% if page.read_time %}
+                    {{ page.read_time }} min{{ page.read_time|pluralize }} read
+                {% endif %}
+            </p>
+
+            <div class="tags">
+                <span class="sr-only">Related work categories</span>
+                {% for tag in page.related_services.all %}
+                    {% include "patterns/atoms/tag/tag.html" with tag_slug=tag.slug tag_name=tag.name tag_link_base="../" %}
+                {% endfor %}
+            </div>
+        </div>
+    </div>
+
+    {% block main_image %}
+        {# The new work page has a dedicated image field but the historical work page does not #}
+    {% endblock %}
+
+    <div class="grid grid--spacer-large streamfield">
+
+        {% block body_streamfield %}
+
+            {% for block in page.body %}
+                {% if block.block_type != 'intro' %}
+                    {% include_block block %}
+                {% endif %}
+            {% endfor %}
+        {% endblock %}
+
+        <div class="work-page-authors">
+            {% for author in page.authors.all %}
```

Comment:
@engineervix would it be a good idea to make the authors a cached property here that are passed the context? I did have a try but it didn't seem to work.

Reply:
it might also mean I could avoid `with author=author.author` which is currently meaning I have to write yaml that fails a prettier check.

Reply:
I've updated the template variables for now to solve the prettier issue.

Reply:
> @engineervix would it be a good idea to make the authors a cached property here that are passed the context? I did have a try but it didn't seem to work.

We could, though we'd have to name it something else, say

```python
    @cached_property
    def authors_list(self):
        return [author.author for author in self.authors.all()]
```

Then in the template we'd have

```django
        <div class="work-page-authors">
            {% for author in page.authors_list %}
                {% if page.first_author == author %}
                    {% include "patterns/molecules/author/author.html" with author=author author_attribution="by" %}
                {% else %}
                    {% include "patterns/molecules/author/author.html" with author=author author_attribution="and" %}
                {% endif %}
            {% endfor %}
        </div>
```

Reply:
Maybe as I've spent a while on this ticket I'll leave it as it is now for speed - it's all working now at least and no complaints from prettier.